detox/src/realms/DetoxSecondaryContext.test.js

```diff
@@ -1,40 +1,35 @@
-// @ts-nocheck
+const { backupProcessEnv, latestInstanceOf } = require('../../__tests__/helpers');
 
-const {
-  latestInstanceOf,
-  throwErrorImpl,
-  withSuspendingMock,
-} = global;
-
-const DETOX_CONFIG_SNAPSHOT_PATH = 'mocked/detox.json';
-const sessionState = {
-  id: 'mocked-session-state',
-};
+describe('DetoxSecondaryContext', () => {
```

Comment:
There is nothing new here regarding my review comments. Please check out `DetoxPrimaryContext.test.js` and `detox/__tests__/helper.js` file to see the motivation of changes.